docs/it/federation_endpoint.rst

```diff
@@ -237,24 +228,20 @@ DEVE essere una richiesta HTTP che usa il metodo GET e lo schema https verso uno
 
 
 .. list-table:: 
-   :widths: 20 20 40 20
+   :widths: 20 60 20
    :header-rows: 1
 
    * - **Claim**
-     - **Tipo**
      - **Descrizione**
      - **Obbligatorio**
    * - **sub**
-     - String
-     - L’entity_id per l’enbtità alla quale è stato inviato il TM.
+     - String. L’entity_id per l’enbtità alla quale è stato inviato il TM.
      - |uncheck-icon|
    * - **id**
-     - String
-     - Identifica il TM
+     - String. Identifica il TM
      - |uncheck-icon|
    * - **iat**
-     - UNIX Timestamp
-     - Identifica quanto il TM è stato emesso. Se non viene specificato **iat** e l’emettitore del TM ha emesso diversi TM con l’**id** specificato nella richiesta all’entità specificata da **sub**, allora si assume che sia l’ultimo.
+     - UNIX Timestamp. Identifica quanto il TM è stato emesso. Se non viene specificato **iat** e l’emettitore del TM ha emesso diversi TM con l’**id** specificato nella richiesta all’entità specificata da **sub**, allora si assume che sia l’ultimo.
```

Comment:
- UNIX Timestamp con la data di emissione del JWT, codificato come NumericDate come indicato in
       `RFC7519 - JSON Web Token (JWT) <https://www.rfc-editor.org/rfc/rfc7519.html>`_